packages/contracts/development/contracts/IRealityETH_ERC20.sol

```diff
@@ -2,9 +2,11 @@
 
 pragma solidity 0.8.20;
 
+// Features removed in v4
```

Comment:
This comment might not be clear after reading it later. It could be clearer if one really says waht was removed. Or maybe its not needed at all?